examples/example_plugin/example_plugin/template_content.py

```diff
@@ -56,6 +56,7 @@ def detail_tabs(self):
             },
         ]
 
+    # TODO: this doesn't seem to be working, only the React components are being rendered as content extensions
```

Comment:
Appears to be broken by the inclusion of `drf-react-template` - see https://github.com/nautobot/nautobot/pull/3285/files#diff-9a2bdd132c5e3a0e6947134b0ae99c641837ad953eb9f5585633bf33e4a60dc9R333

Reply:
`drf-react-template` was added by me at the last minute to get the JSON Schema Forms stuff working. It changes all the API endpoints and the way it does it is not ultimately what we want. We'll have to modify it.